File: AdvancedLevel_C++/1005. Spell It Right (20).cpp
#include <iostream>
using namespace std;
int main() {
    string a;
    cin >> a;
    int sum = 0;
    for (int i = 0; i < a.length(); i++)
        sum += (a[i] - '0');
    string s = to_string(sum);
    string arr[10] = {"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"};
    cout << arr[s[0] - '0'];
    for (int i = 1; i < s.length(); i++)
        cout << " " << arr[s[1] - '0'];
    return 0;
}

File: AdvancedLevel_Java/1005 Spell It Right (20).java
import java.io.IOException;
import java.io.InputStream;

public class Main {
    static String[] words_table = new String[]{"zero",
            "one", "two", "three", "four", "five",
            "six", "seven", "eight", "nine"};

    public static void main(String[] args) throws IOException {
        InputStream reader = System.in;
        int input, sum = 0;
        while ((input = reader.read()) != '\n') {
            sum += input - 48;
        }
        char[] chars = (sum + "").toCharArray();
        for (int i = 0; i < chars.length; i++) {
            System.out.print(words_table[chars[i] - 48]);
            if (i == chars.length - 1) {
                System.out.println();
            } else {
                System.out.print(" ");
            }
        }
    }
}